{{cookiecutter.project_slug}}/containers/Dockerfile

```
@@ -0,0 +1,82 @@
+# ref: https://github.com/mamba-org/micromamba-docker/blob/main/Dockerfile
+{% if cookiecutter.add_containers == "Docker" %}
+FROM condaforge/mambaforge:latest
+{% else %}
```

```suggestion
{% elif cookiecutter.add_containers == "Podman" %}
```

it would be good to have podman explicitly here instead of else 
probably it would not change too much the result .. as if it is none it would be removed anyway .. 
but if if we have any new entry for add_containers ... have it explicitly would make things more clear